rest/src/resolve.cpp

```diff
@@ -16,9 +15,14 @@ unsigned Resolve::run()
         throw rest::Error(ret.error());
     }
 
-    auto strIdPrt = m_request.queryArg<std::string>("id");
+    auto        strIdPrt = m_request.queryArg<std::string>("id");
```

Comment:
you removed id from the url definition... why are you trying to get it in code? This id will never exists :)

Reply:
we have two mapping and one of them(GET) with id

Reply:
I'm not sure if it work at all... possible you can't map different urls to 1 callback